mod/beacon/validator/block_builder.go

```diff
@@ -303,7 +304,12 @@ func (s *Service[
 	))
 
 	// Set the graffiti on the block body.
-	body.SetGraffiti(bytes.ToBytes32([]byte(s.cfg.Graffiti)))
+	sizedGraffiti := bytes.ExtendToSize([]byte(s.cfg.Graffiti), bytes.B32Size)
```

Comment:
ideally we would like to change `defaultGraffiti` to this, but that would not be a valid UTF-8 string and it would fail validation.